cmd/lima-guestagent/daemon_linux.go

```diff
@@ -40,17 +28,11 @@ var daemonCommand = &cli.Command{
 }
 
 func daemonAction(clicontext *cli.Context) error {
-	socket := clicontext.String("socket")
-	if socket == "" {
-		return errors.New("socket must be specified")
-	}
+	socket := "/run/lima-guestagent.sock"
 	tick := clicontext.Duration("tick")
 	if tick == 0 {
 		return errors.New("tick must be specified")
 	}
-	if os.Geteuid() == 0 {
```

Comment:
Maybe reverse the check and throw an error when **not** running as root?

Reply:
That's a good idea

Reply:
done